src/selectors/contents.js

```diff
@@ -64,21 +82,28 @@ export const getDirectories = createSelector([getContentsPerDirectories], conten
   ),
 )
 
-export const getFavoritesContents = createSelector([getAllContents], contents =>
-  contents.filter(c => c.isFavorite),
+export const getAugmentedFavoriteContents = createSelector(
+  [getAllContents, getFavorites],
+  (contents, favorites) =>
+    getAugmentedContents(contents.filter(content => favorites[content.id]), favorites),
```

Comment:
See the next comment about `typeof` and `undefined`

Reply:
Can't really work here : A content can be put into favorite and removed, and I think `favorites[content.id]` will be equal to false in that case